dataeng/jobs/analytics/SnowflakeReplicaImportFromS3.groovy

```diff
@@ -1,37 +1,91 @@
+                INCLUDE: '--include ["catalogs_catalog"]',
+            ],
+            ECOMMERCE: [
+                CLUSTER_NAME: 'ImportEcommerceReadReplicaSnowflakeFromS3',
+                SCHEMA: 'ECOMMERCE_RAW',
+                SCRATCH_SCHEMA: 'ECOMMERCE_SCRATCH_LOADING',
+                DATABASE: 'ecommerce',
+                INCLUDE: '--include ["basket_line", "basket_basket_vouchers", "catalogue_historicalproduct", "catalogue_product", "catalogue_productattributevalue", "django_session", "ecommerce_user", "invoice_invoice", "offer_condition", "offer_offerassignment", "offer_offerassignmentemailtemplates", "order_line", "refund_refund"]',
+            ],
+        ]
+
+        jobConfigs.each { appName, jobConfig ->
+            dslFactory.job("snowflake-${appName.toLowerCase()}-read-replica-import-from-s3") {
                 logRotator common_log_rotator(allVars)
-                parameters common_parameters(allVars, db_config)
+                parameters common_parameters(allVars, jobConfig)
                 parameters {
-                    stringParam('RUN_DATE', db_config.get('RUN_DATE', allVars.get('RUN_DATE', 'today')), 'Run date for the job. A string that can be parsed by the GNU coreutils "date" utility.')
-                    stringParam('OVERWRITE', db_config.get('OVERWRITE', allVars.get('OVERWRITE')), 'Set to: --overwrite if you want to enable overwrite.')
-                    stringParam('SNOWFLAKE_CREDENTIALS', db_config.get('SNOWFLAKE_CREDENTIALS', allVars.get('SNOWFLAKE_CREDENTIALS')), 'The path to the Snowflake credentials file.')
-                    stringParam('WAREHOUSE', db_config.get('WAREHOUSE', allVars.get('WAREHOUSE')), '')
-                    stringParam('ROLE', db_config.get('ROLE', allVars.get('ROLE')))
-                    stringParam('SNOWFLAKE_DATABASE', db_config.get('SNOWFLAKE_DATABASE', allVars.get('SNOWFLAKE_DATABASE')))
-                    stringParam('SCHEMA', db_config.get('SCHEMA', allVars.get('SCHEMA')), 'Schema')
-                    stringParam('SCRATCH_SCHEMA', db_config.get('SCRATCH_SCHEMA', allVars.get('SCRATCH_SCHEMA')), 'Scratch schema name - temporary loading location.')
-                    stringParam('DATABASE', db_config.get('DATABASE'), 'Name of MySQL database to copy from.')
-                    stringParam('INCLUDE', db_config.get('INCLUDE'), 'List of regular expressions matching table names that should be copied.')
-                    stringParam('EXCLUDE', db_config.get('EXCLUDE'), 'List of regular expressions matching table names that should not be copied.')
+                    stringParam('RUN_DATE', 'today', 'Run date for the job. A string that can be parsed by the GNU coreutils "date" utility.')
+                    stringParam('OVERWRITE', '--overwrite', 'Set to: --overwrite if you want to enable overwrite.')
+                    stringParam('SNOWFLAKE_CREDENTIALS', allVars.get('SNOWFLAKE_CREDENTIALS'), 'The path to the Snowflake credentials file.')
```

Comment:
If you're moving everything else in here, is there a reason not to move SNOWFLAKE_CREDENTIALS as well?   Isn't it  just an S3 path?

Reply:
It is just an S3 path, But I just wasn't sure if it was too-much-information for this open source repo.